core/Pos/Util/Mnemonics.hs

```diff
@@ -0,0 +1,489 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- Taken and slightly modified from https://github.com/input-output-hk/postvend-app/blob/master/src/Mnemonics.hs
+
+module Pos.Util.Mnemonics
+       (
+         -- * Data types
+         Entropy
+       , Mnemonic
+       , Seed
+
+         -- * Entropy encoding and decoding
+       , toMnemonic
+       , fromMnemonic
+
+         -- * Helper functions
+       , getBits
```

Comment:
I don't think it's needed to be exported. The whole `Mnemonics.hs` from postvend-app was written without caring about style and whatsoever. So i don't know. Maybe we'll need to refactor it later.